datasets/ro/onpcsb_sanctions/crawler.py

```diff
@@ -0,0 +1,88 @@
+import csv
+from typing import Dict
+from zavod import Context, helpers as h
+from typing import List
+
+
+DATE_FORMATS = ["%m/%d/%Y", "%Y"]
+
+
+def parse_date(text: str) -> List[str]:
+    # add a function to handle multiple entires
+    if not text:
+        return []
+    date = h.parse_date(text, DATE_FORMATS)
+    return date
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    full_name = row.pop("name")
+    other_name = row.pop("other name")
+    birth_date_1 = parse_date(row.pop("date of birth"))
```

Comment:
you might not need the parse_date function if you 

```suggestion
    birth_date_1 = h.parse_date(row.pop("date of birth") or None, FORMATS)
```